src/net_processing.cpp

```diff
@@ -502,6 +532,13 @@ void FindNextBlocksToDownload(NodeId nodeid, unsigned int count, std::vector<con
 
 } // namespace
 
+// Returns true for outbound peers, excluding manual connections, feelers, and
+// one-shots
+bool IsOutboundPeer(const CNode *node)
```

Comment:
This name is really deceiving. Maybe IsOutboundDisconnectionCandidate?

Reply:
done